encord/user_client.py

```diff
@@ -142,14 +146,14 @@ def get_project(self, project_hash: str) -> Project:
         # not full access, that is implied by get_ontology method
         ontology_hash = orm_project["ontology_hash"]
         orm_ontology = querier.basic_getter(OrmOntology, ontology_hash)
-        project_ontology = Ontology(querier, orm_ontology)
+        project_ontology = Ontology(querier, orm_ontology, self._api_client)
```

Comment:
Not entirely sure about adding the api_client to every instantiation of the Ontology class

Reply:
It's totally fine - this is just a REST client with basically no "owned" state